standardize.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# exit immediately when any command fails
+set -e
+
+go build ./... 
+golangci-lint run # this will fail if the code is formatted incorrectly, see ./.golangci.yml
+go test ./...
+go mod tidy # this may generate some changes...
```

Comment:
It looks like a[ check flag ](https://github.com/golang/go/issues/27005)is planned for `go mod tidy` but missed the cut off for golang 1.18 :(